doc/installation_and_administration_guide/README.md

```diff
@@ -1,6 +1,7 @@
 #Installation and Administration Guide
 
 * [Introduction](./introduction.md)
+* [Hardware requirements](./hw_requirements.md)
```

Comment:
Fixed in <https://github.com/telefonicaid/fiware-cygnus/commit/69bad2f614891f36366db42e5b3636c2d0b591d0>